### ISSUE TITLE
Accessibility audit: reconsider contrast of grey cube image

### ISSUE THREAD

**Issue by nlhkabu** [pypa/warehouse#3924]
From our accessibility audit:

> There is a color contrast issue with the cube shaped figure under the headings "trending projects" and
"new releases". There is nothing in any guideline about contrasts when it comes to images and logos,
and it may seem picky since the figure does not communicate any vital information on the page.
However, if a user sees something on the page but can't make out what meaning it conveys, it is
possible that the user gets frustrated because the user feels like he or she is missing out on
information. This scenario is especially true when it comes to visually impaired users because it is
common that web pages exclude them from a full user experience.

I'm not sure how we should address this - we do have a blue cube image, but it might look a bit heavy...

**Comment by nlhkabu** [comment]
I am going to close this one as this contradicts the [WGAG guidelines](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html), which excludes any images that are not "required for understanding" from color contrast requirements:

"The graphic is for aesthetic purposes that does not require the user to see or understand it to understand the content or use the functionality."

We can reopen this if it proves to be a problem raised when conducting accessibility user testing.
